actors/builtin/manifest/manifest.go

```diff
@@ -0,0 +1,49 @@
+package manifest
+
+import (
+	"context"
+	"fmt"
+
+	adt8 "github.com/filecoin-project/specs-actors/v8/actors/util/adt"
+
+	"github.com/ipfs/go-cid"
+)
+
+type Manifest struct {
```

Comment:
If you keep this structure this should move to the migration package as that's the only code that uses (ever will use?) this type.

Reply:
I think we want to keep it for completeness.

Note that the constituents (ManifestData, ManifestEntry) are part of the system actor state now, so they have to be here.
So I'd rather keep everything here as oppose to bifurcating and having half the stuff here and the outer struct in migrations.

It also gives us a nice entry point for go code that wants to read manifests.

Reply:
Also note, we'll be using it in every subsequent migration from now on.

Reply:
It's confusing to keep it here since the data is not part of the system actor state. It would be clear to put it in the migration package.  The fact that it is in all subsequent migrations strengthens the argument for moving it there.